_includes/posts/_maintenance-table.md

```diff
@@ -1,7 +1,8 @@
-| 0.8  | 2013-02-19 | 2014-03-19  | 2015-12-31 |
-| 0.9  | 2014-03-19 | 2015-02-16  | 2016-02-28 |
-| 0.10 | 2015-02-16 | 2016-02-29  | 2017-02-28 |
-| 0.11 | 2015-07-12 | after v0.13 | TBA        |
-| 0.12 | TBA        | after v0.14 | TBA        |
+| 0.8  | 2013-02-19   | 2014-03-19  | 2015-12-31 |
+| 0.9  | 2014-03-19   | 2015-02-16  | 2016-02-28 |
+| 0.10 | 2015-02-16   | 2016-02-29  | 2017-02-28 |
+| 0.11 | 2015-07-12   | after v0.13 | TBA        |
+| 0.12 | *2016-02-xx* | after v0.14 | TBA        |
+| 0.13 | *2016-07-xx* | after v0.15 | TBA        |
```

Comment:
In the explainer text we say we aim for releases every 6-7 months, maybe that's enough as is. If we have dates I would tend to put `~2016-07-31` because it's the midpoint of 6-7 months.